ear/fileio/utils.py

```diff
@@ -26,6 +36,18 @@ def openBw64(filename, mode='r', **kwargs):
 
 
 def openBw64Adm(filename, fix_block_format_durations=False):
+    """Open a BW64 ADM file for reading. This automatically parses the ADM, and
+    adds common definitions and information from the CHNA chunk, accessed
```

Comment:
The way this is worded suggests that the common definitions are in the CHNA chunk.  Something from my poking around seemed to show me a load of common definitions, perhaps all of them, and suggested to me that it's more along these lines:
"This automatically parses the ADM, adds the information from the CHNA chunk, and adds the common definitions, ..."    But I don't know from where the common defs are added - somewhere they seemed to appear as if by magic.

Reply:
Fixed, but in a slightly different way.

The common definitions are added by the XML parsing functions, I just tend to mention it at higher levels too to minimise surprise.